frontend/hub/namespaces/PageInputDynamicFields.tsx

```diff
@@ -0,0 +1,67 @@
+import { Button, Flex, FlexItem, TextInput } from '@patternfly/react-core';
+import { PlusCircleIcon, TrashIcon } from '@patternfly/react-icons';
+import { useFieldArray, useFormContext, Controller } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { PageFormSection } from '../../../framework/PageForm/Utils/PageFormSection';
+
+export function PageInputDynamicFields() {
```

Comment:
This name will be updated. Just to have an idea what direction this work was going.